lib/index.js

```diff
@@ -10,7 +10,7 @@ var internals = {
 internals.store.lookUp = function (obj, method) {
   var i = this.length;
   var lookup;
-  while (i-- && !lookup) {
+  while (!lookup && i--) {
```

Comment:
The only reason people use `break` is because they have the `for` loop on speed dial and don't think of using a `while` loop.